releases/4.0.0-beta.2.md

```diff
@@ -0,0 +1,59 @@
+---
+title: TiDB 4.0.0-beta.2 Release Notes
+category: Releases
+---
+
+# TiDB 4.0.0-Beta.2 Release Notes
+
+Release date: March 13, 2020
+
+TiDB version: 4.0.0-beta.2
+
+TiDB Ansible version: 4.0.0-beta.2
+
+## TiDB 4.0.0-beta.2 Compatibility Changes
+
++ Tools
+    - TiDB Binlog
+        - Fix the issue that the system returns an error and exits when `disable-dispatch` and `disable-causality` are configured in Drainer [#915](https://github.com/pingcap/tidb-binlog/pull/915)
+
+## New Features
+
++ TiKV
+    - Support persisting the dynamically updated configuration into the hardware disk [#6684](https://github.com/tikv/tikv/pull/6684)
+
++ PD
+    - Support persisting the configuration of dynamic updates into hard drives [#2153](https://github.com/pingcap/pd/pull/2153)
+
++ Tools
+    - TiDB Binlog
+        - Support the bidirectional data replication between TiDB clusters [#879](https://github.com/pingcap/tidb-binlog/pull/879) [#903](https://github.com/pingcap/tidb-binlog/pull/903)
+    - TiDB Lightning
+        - Support the TLS configuration [#40](https://github.com/tikv/importer/pull/40) [#270](https://github.com/pingcap/tidb-lightning/pull/270)
+    - TiCDC
+        - Initial release of the change data capture (CDC), providing the following features:
+            - Support capturing changed data from TiKV
+            - Support replicating the changed data to MySQL protocol-compatible database, with an eventual consistency guarantee
+            - Support replicating the changed data to Kafka, with either an eventual consistency guarantee or row-level order guarantee
```

Comment:
Please check with PM whether `an eventual consistency guarantee` and `row-level order guarantee` has the `either...or` relationship.